.github/workflows/ci.yml

```diff
@@ -345,6 +347,9 @@ jobs:
           - component:  Distributed
             suite: DistributedSystems
             maven-modules: ${{ needs.setup-unit-tests.outputs.ZEEBE_DISTRIBUTED_MODULES }}
+          - component: Client
+            suite: Clients
```

Comment:
For the other matrix jobs, `suite` is `CoreFeatures` or `DataLayer` etc indicating to me that it should mean team names: should it be `CamundaEx` here?